rockwire.yaml

```
@@ -0,0 +1,425 @@
+openapi: 3.0.0
+info:
+  title: Rockwire Platform APIs
```

```suggestion
  title: Rockwire Platform API
```
My previous suggestion had a typo. Sorry.